pymc3/tests/test_sampling.py

```diff
@@ -56,7 +57,7 @@ def test_parallel_sample_does_not_reuse_seed(self):
         random_numbers = []
         draws = []
         for _ in range(2):
-            np.random.seed(1)  # seeds in other processes don't effect main process
+            np.random.seed(1)  # seedsa in other processes don't effect main process
```

Comment:
```suggestion
            np.random.seed(1)  # seeds in other processes don't effect main process
```

Reply:
Also should be "affect", not "effect."

Reply:
Fixed both of these and pushed to the MR.